x/qgb/keeper/query_data_commitment.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	"context"
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+func (k Keeper) DataCommitmentConfirm(c context.Context, request *types.QueryDataCommitmentConfirmRequest) (*types.QueryDataCommitmentConfirmResponse, error) {
+	addr, err := sdk.AccAddressFromBech32(request.Address)
+	if err != nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "address invalid")
+	}
+	return &types.QueryDataCommitmentConfirmResponse{Confirm: k.GetDataCommitmentConfirm(sdk.UnwrapSDKContext(c), request.Commitment, addr)}, nil
+}
+
+func (k Keeper) DataCommitmentConfirmsByValidator(ctx context.Context, request *types.QueryDataCommitmentConfirmsByValidatorRequest) (*types.QueryDataCommitmentConfirmsByValidatorResponse, error) {
+	addr, err := sdk.AccAddressFromBech32(request.Address)
+	if err != nil {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "address invalid")
+	}
+	return &types.QueryDataCommitmentConfirmsByValidatorResponse{Confirms: k.GetDataCommitmentConfirmsByValidator(sdk.UnwrapSDKContext(ctx), addr)}, nil
+}
+
+func (k Keeper) DataCommitmentConfirmsByCommitment(ctx context.Context, request *types.QueryDataCommitmentConfirmsByCommitmentRequest) (*types.QueryDataCommitmentConfirmsByCommitmentResponse, error) {
```

Comment:
do we not have CI linters for line length? This is a bit of nit pick, but pretty sure these lines are too long. I forget what the limit is tbh

Reply:
Locally, my linter config complains. However, since it is passing GitHub one, I thought it's alright.